### PR TITLE
bump up version of netcore framework to 1.1.0

### PR DESCRIPTION
**Pull request by gcristofol**
Bump up version of netcore framework to 1.1.0 across all projects

### CODE REVIEW

**Comment by LammyL** [comment]
i am wondering what the repercussions are with leaving this as netstandard1.6 rather than bumping up the library?  It would seem to me that changing it to the latest version would prevent older apps from running this library or linking to this library.

**Comment by gcristofol** [comment]
Probably I could add netstandard on the top level project.json. The only incompatibiliy I did find was dotnet-test-xunit so (limited to the environment I have for testing) it should run just fine.

File: src/Swashbuckle.Swagger/project.json [hunk 1]
```diff
@@ -31,6 +34,12 @@
 
   "frameworks": {
     "net451": {},
+    "netcoreapp1.1.0": {
```
Comment (alexvaluyskiy) [inline src/Swashbuckle.Swagger/project.json]: It is not necessary to add netcoreapp to library projects

**Comment by timotyweis** [comment]
Please update also the following references in `SwaggerUI`
- "Microsoft.AspNetCore.StaticFiles": "1.0.0" --> 1.1.0
- "Microsoft.Extensions.FileProviders.Embedded": "1.0.0" --> 1.1.0

**Comment by timotyweis** [comment]
Please ensure that also `Microsoft.AspNetCore.StaticFiles` and `Microsoft.Extensions.FileProviders.Embedded` are moved to 1.1.0.

**Comment by dewrox** [comment]
Is there an ETA for this to be resolved?  Without Core 1.1 integration being complete, uptake of this project will be hampered.

**Comment by jarroda** [comment]
As has been mentioned before, it is not necessary for this library to be updated to use it in a 1.1 app.  We're using it in multiple apps today without any issues.

**Comment by dewrox** [comment]
Has anyone used this (as Core 1.0) with Identity Server 4.0 (Core 1.1.0) and had the Authorize work?  Identity Server 4.0 group says it is a client issue... the client in this case is Swagger.

**Comment by domaindrivendev** [comment]
Re-iterating @jarroda's comments. This project depends on Core 1.0 at a minimum but absolutely works if you upgrade to 1.1.

**Pull request closed without merging**